src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,16 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("completeOrder was called");
```

Comment:
The log message 'completeOrder was called' is not very informative. Consider including the userId parameter in the log message to provide more context. For example: `logger.info("completeOrder was called for userId: {}", userId);` This would align with the checklist's recommendation to make log messages informative.

Reply:
Changed